src/garam-masala.md

```diff
@@ -0,0 +1,38 @@
+# Garam Masala
+
+A spice mixture for most Indian dishes
+
+## Ingredients
+
+- 4 Green Cardamom
+- 2 Black Cardamom
+- 1 tbsp Coriander
+- 2 Cloves
+- 1 tbsp Cumin
+- 1 tbsp Fennel seeds
+- 5/6 Bayleaf
+- 5/6 Black pepper pods
+- 1 Star anise (Optional)
+- 0.25 Mace (Optional)
```

Comment:
0.25 of what unit?

Also, maybe I am just ignorant but I've never heard of mace before. Thought for a second we were talking about pepper because mace is a brand of pepper spray.

If it is common and I am just an idiot, ignore me. But otherwise a short description of what it is may be useful. Looks like it comes from the nutmeg seed, so maybe nutmeg could be mentioned as well.

Reply:
Yes, Mace is created from the Nutmeg seed and it generally comes in a fixed size. I meant 0.25 of that one piece of Mace. Not sure what would be the best way to write this.